src/screenipy.py

```diff
@@ -79,14 +85,50 @@ def initExecution():
         Utility.tools.clearScreen()
         return initExecution()
 
+
+def isTradingTime():
+    curr = datetime.now(pytz.timezone('Asia/Kolkata'))
+    openTime = curr.replace(hour=9, minute=15)
+    closeTime = curr.replace(hour=15, minute=30)
+    isTradingTime = ((openTime <= curr <= closeTime) and (
+        0 <= curr.weekday() <= 4))
+
+
+def saveStockData():
+    global stockDict
+    today_date = date.today().strftime("%d%m%y")
+    cache_file = "stock_data_" + str(today_date) + ".pkl"
+    if not os.path.exists(cache_file):
```

Comment:
Need to call **`deleteStockMethod`** here before saving pickle so the user doesn't need to manually erase previously pickled files.
```suggestion
    cache_file = "stock_data_" + str(today_date) + ".pkl"
    configManager.deleteStockMethod()
    if not os.path.exists(cache_file):
```

Reply:
Here, ```configManager.deleteStockMethod``` will delete stock data every time. File will written on disk with same data multiple times.  According to me, ```configManager.deleteStockMethod()```` should not be called there.

Reply:
Yeah, We need a method that deletes all other `stock_data*.pkl` except the file with given filename.

Reply:
Done in 35496b8